.gitignore

```diff
@@ -21,3 +21,7 @@
 
 # virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
 hs_err_pid*
+/target/
+/.classpath
+/.project
+/.settings/
```

Comment:
Could you please add a comment like for the other sections?
Do you need the initial slash, these look like absolute paths